### PR TITLE
Feature/add user pantry info

### PR DESCRIPTION
**Pull request by Alynn022**
PR Template:

Is this a fix or a feature?
- [ ] Bug fix
- [x] Feature
- [ ] Refactor
 
Describe the feature - fix - refactor: create method for comparing the pantry ingredient info and recipe ingredient info 

How was this tested?
- [ ] open index.html
- [x] localhost
- [ ] terminal

Where should the reviewer start - test: pantry.js and pantry-test.js 

Additional comments:

### CODE REVIEW

File: src/classes/Pantry.js [hunk 1]
```diff
@@ -1,46 +1,69 @@
...
-  //     })
-  //     return allUserIngredientAmt;
-  //   }, [])
-  //   return userIngredientsAmt
-  // }
-
-  determineAmountOfIngredientsUserHas() {
-    let userIngredientAmt = this.shelf.map(ingredient => {
-      return ingredient.amount 
+  showPantryIngredientInfo() {
```
Comment (StevieAmb) [inline src/classes/Pantry.js:9]: Pushing in the information in the form of an object was a great idea!

File: src/classes/Pantry.js [hunk 2]
```diff
@@ -1,46 +1,69 @@
...
-    let userIngredientAmt = this.determineAmountOfIngredientsUserHas(recipe);
-    let userMessage = "";
-    console.log(userIngredientAmt);
-    userIngredientAmt.forEach(amount => {
-      recipe.ingredients.forEach(ingredient => {
-        if(amount < ingredient.quantity.amount || !userIngredientAmt.includes(ingredient.quantity.amount)) {
-          userMessage = "You don't have enough ingredients to cook this recipe"
-        } else {
-          userMessage = "Let's get cookin', good lookin";
+    let userIngredientAmt = this.showPantryIngredientInfo();
```
Comment (StevieAmb) [inline src/classes/Pantry.js:22]: Nested for loooooop!!!!

File: src/classes/Pantry.js [hunk 3]
```diff
@@ -1,46 +1,69 @@
...
+      if (match === false) {
+       return "You don't have enough ingredients to cook this recipe";
+      
+      }
+    } 
+    return "Let's get cookin', good lookin!"
   }
 }
 
+// determineIfEnoughIngredientsForRecipe(recipe) {
```
Comment (StevieAmb) [inline src/classes/Pantry.js:42]: One day, we'll figure this out.

**Review by StevieAmb — APPROVED** [review]

**Review by lschnell8 — APPROVED** [review]
Nice job getting this working. I have some refactor suggestions that covers more tasks with less code that I'd like to show you.

File: src/classes/Pantry.js (hunk 2 above)
Comment (lschnell8) [reply src/classes/Pantry.js:22]: We have a fix for that! :)

File: src/classes/Pantry.js (hunk 3 above)
Comment (lschnell8) [reply src/classes/Pantry.js:42]: Oh we got this!

File: src/classes/Pantry.js (hunk 1 above)
Comment (lschnell8) [reply src/classes/Pantry.js:9]: Heck yea!

**Pull request merged**